packages/client/src/forms/register/fieldMappings/birth/query/registration-mappings.ts

```diff
@@ -171,23 +172,21 @@ export const certificateDateTransformer =
     window.__localeId__ = prevLocale
   }
 
-const convertToLocal = (
-  mobileWithCountryCode: string,
-  country: string,
-  codeReplacement?: string
-) => {
+const convertToLocal = (mobileWithCountryCode: string, country: string) => {
   /*
    *  If country is the fictional demo country (Farajaland), use Zambian number format
    */
   const countryCode =
     country.toUpperCase() === 'FAR' ? 'ZMB' : country.toUpperCase()
 
+  const data =
+    allCountries.find(
+      (countryData) => countryData.iso2 === countryCode.slice(0, 2)
+    ) || allCountries[allCountries.length - 3]
+
```

Comment:
Let's make this a bit more readable so the code reader doesn't have to open up allCountries list to understand why the 3rd to the last item is selected

```suggestion
  const defaultCountryZambia = allCountries[allCountries.length - 3]
  const data =
    allCountries.find(
      (countryData) => countryData.iso2 === countryCode.slice(0, 2)
    ) || defaultCountryZambia

```